docs/INSTALL.md

```diff
@@ -11,12 +11,14 @@ libcurl from source code.
 
 You can download and install curl and libcurl using the [vcpkg](https://github.com/Microsoft/vcpkg/) dependency manager:
 
+
     git clone https://github.com/Microsoft/vcpkg.git
     cd vcpkg
     ./bootstrap-vcpkg.sh
     ./vcpkg integrate install
     vcpkg install curl[tool]
 
+
```

Comment:
Why the extra newlines? I see no visual difference on github with or without them?

Reply:
This could be a mistake. I will fix it.